src/main/avro/KafkaRecord.avsc

```diff
@@ -0,0 +1,15 @@
+{"namespace": "com.teragrep.cfe_39.avro",
+  "type": "record",
+  "name": "SyslogRecord",
+  "fields": [
+    {"name": "timestamp", "type": "long"},
+    {"name": "message", "type": "string"},
```

Comment:
could the payload be the last?

Reply:
Solved in commit ea10fa910e39681601e1a793dc729638932bf8b2